addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -222,6 +222,11 @@
             run: () => null,
         },
         ...addExistingField('email_cc', 'text', 'Test conditional visibility', false, {visibility: CONDITIONALVISIBILITY, condition: 'odoo'}),
+        {
+            content: "Ensure that the description has correctly been added on the field",
```

Comment:
Note: you can't remove / toggle off the description of this field. I guess it's a bug but not really, since we force a description in the JS for this field especially, we can maybe close our eyes on this one.
It appears in Odoo 16 as well so it's not related to this PR anyway.